### PR TITLE
Add urgent needs to menu card

### PR DESCRIPTION
**Pull request by andyinabox**
Now urgent needs are displayed in main menu

Screenshot 2020-06-01 12:29:38

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -202,6 +202,7 @@
                 .setPopup(new mapboxgl.Popup().setMaxWidth('250px').setHTML(markerHtml))
                 .addTo(map);
 
+              const urgentNeed = item.gsx$urgentneed.$t ? `<h3 style="color: #f00; font-size: 80%">Urgent Need: ${item.gsx$urgentneed.$t}</h3>` : ''
```
Comment (jdalt) [inline index.html]:
```suggestion
              const urgentNeed = location.urgentNeed ? `<h3 style="color: #f00; font-size: 80%">Urgent Need: ${location.urgentNeed}</h3>` : ''
```

Comment (jdalt) [reply index.html]: Sorry, this stuff is pretty messy. I've been writing it since 10:30pm last night.

**Comment by andyinabox** [comment]
NP! better messy in the wild than perfect in your head :)

**Comment by jdalt** [comment]
Love it!

**Pull request merged**